rllib/algorithms/algorithm.py

```diff
@@ -670,6 +672,36 @@ def setup(self, config: AlgorithmConfig) -> None:
                     "offline.offline_evaluator::OfflineEvaluator"
                 )
 
+        self.trainer_runner = None
+        if self.config.get("_enable_trainer_runner", False):
+            # TODO (Kourosh/Avnishn) worker should not give us observation/action
+            # space. It should be part of the global config.
+            worker = self.workers.local_worker()
+
+            # TODO: The constructor is not clean and comprehensive.
```

Comment:
as a best practice, can we all agree to write TODOs in the format of TODO(<username>).
this allows everyone to understand who came up with this TODO, in case there are questions down the road about the relevance, for example.

Reply:
yes I have got a bit lazy on sticking to this convention :)